### PR TITLE
Example dotnet

### PR DESCRIPTION
**Pull request by rberlind**
I'm adding a .NET Core runtime package and custom dotnet stager that can be used to stage .NET Core and ASP.NET Core applications onto Apcera.  This is in connection with a project Jared Skinner recently had me do for a customer who is interested in running ASP.NET Core apps on Apcera.  Jared would also like me to record a YouTube video about this and write a blog post that would be posted to Apcera's website; but it only makes sense to do that if we can refer viewers/readers to a public repository containing the referenced code.

### CODE REVIEW

File: example-dotnet/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+### Start Commands
+When staging source code, we set the start command to `dotnet restore ; dotnet run`. If running dotnet restore each time you start the app takes too long, I recommend that you stage your published DLLs instead. When staging published .NET DLLs, we set the start command to `dotnet <app_name>.dll` where \<app_name\> matches the prefix of the .deps.json file.
+
+### Why No Dotnet Dependency Is Included
+In the first version of the dotnet stager, I actually included the dotnet-core dependency provided by the dotnet-core-sdk-1.0.1-ubuntu-14.04 runtime package. However, the first customer who tested the stager wanted the flexibility to base their .NET apps on various standard .NET Docker images as well as on runtime packages like the one I had created. To give customers more flexibility, I removed the dotnet-core dependency from the stager. This does have the consequence, however, that customers must specify their own dotnet dependency with the --depends-on or -do option of the `apc app create` command or in the package_dependencies section of a continuum.conf application manifest.
```
Comment (tstatler) [inline example-dotnet/README.md]: nit: suggest wrapping `--depends-on` and `-do` command line options in backticks to apply code font.

**Comment by rberlind** [comment]
Thanks Tim.  Done.
What would be the next steps to get this pull request merged?

**Comment by tstatler** [comment]
Just merge away. Protocol around the sample-apps repo is pretty loosey goosey. Great readme, btw!

**Comment by rberlind** [comment]
Thanks Tim

**Pull request merged**